### DISCUSSION TITLE
Support user uploading images in steganography v2

### DISCUSSION THREAD

**Discussion by yisiliu** [DimensionDev/Maskbook#2155] [Ideas]
Shall we store the images in the local storage (indexdb) or shall we force the users to upload them onto an image hosting service(e.g. imgur)? If the former, can we wrap it in the db backup file and how? If the latter, shall we host our own objective storage (e.g. AWS S3) or directly use services like imgur?

**Comment by septs** [comment]
https://github.com/DimensionDev/Maskbook/issues/2116
